src/Simplex/Chat.hs

```diff
@@ -3294,14 +3283,23 @@ processAgentMsgSndFile _corrId aFileId msg =
             liftIO $ updateCIFileStatus db user fileId status
             lookupChatItemByFileId db vr user fileId
           toView $ CRSndFileProgressXFTP user ci ft sndProgress sndTotal
-        SFDONE sndDescr rfds -> do
+        SFDONE sndDescr [] -> do
+          -- should not happen
+          logError "File sent without receiver descriptions"
+          withStore' $ \db -> setSndFTPrivateSndDescr db user fileId (fileDescrText sndDescr)
+          withAgent (`xftpDeleteSndFileInternal` aFileId)
```

Comment:
this might break sending files to empty groups. It may be suboptimal that they are being uploaded at all (not necessary to fix as notes solve this) but with this it may stop working?